--- /usr/share/perl5/Net/GitHub/V3/Issues.pm.orig	2015-01-19 15:58:08.000000000 +0000
+++ /usr/share/perl5/Net/GitHub/V3/Issues.pm	2015-01-19 16:04:33.000000000 +0000
@@ -40,6 +40,17 @@
     return $self->query($u);
 }
 
+sub comments {
+    my $self = shift;
+    my $issue = shift;
+    my $args = @_ % 2 ? shift : { @_ };
+
+    my $u = "/repos/" . uri_escape($self->u) . "/" . uri_escape($self->repo)
+        . '/issues/' . uri_escape($issue) . '/comments';
+    $u .= '?since=' . uri_escape($args->{since}) if exists $args->{since};
+    return $self->query($u);
+}
+
 ## build methods on fly
 my %__methods = (
 
@@ -48,7 +59,6 @@
     update_issue => { url => "/repos/%s/%s/issues/%s", method => 'PATCH', args => 1 },
 
     ## http://developer.github.com/v3/issues/comments/
-    comments => { url => "/repos/%s/%s/issues/%s/comments" },
     comment  => { url => "/repos/%s/%s/issues/comments/%s" },
     create_comment => { url => "/repos/%s/%s/issues/%s/comments", method => 'POST',  args => 1 },
     update_comment => { url => "/repos/%s/%s/issues/comments/%s", method => 'PATCH', args => 1 },
